containers/node-bootstrap-container/tests/helper-test.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// eslint-disable-next-line node/no-unpublished-import
+const path = require('path');
+const createGhIssue = require('../../node-bootstrap-container/create-gh-issue');
+const createFolderNamePath = path.resolve(__dirname, '../../node-bootstrap-container/create-folder-name');
+const nock = require('nock');
+const snapshot = require('snap-shot-it');
+const {describe, it} = require('mocha');
+const assert = require('assert');
+const cp = require('child_process');
+const { Octokit } = require('@octokit/rest');
+
+nock.disableNetConnect();
+
+describe('helper functions', () => {
+    it('should return the API path without the version, with dashes', async () => {
+      assert.deepStrictEqual(cp.execSync(
+        `node ${createFolderNamePath} 'google.cloud.kms.v1'`).toString('utf-8'),
+        'google-cloud-kms\n'
+      );
+    });
+
+  
+      it('should open an issue on a given repo', async () => {
+       process.env.MONO_REPO_NAME = 'google-cloud-node';
```

Comment:
it would be good to cleanup environment variable after the test run, so that the behavior doesn't carry over to the next test, I believe you could do something like:

`sinon.stub(process.env, 'FOO').value('bar');`